packages/compass-preferences-model/src/storage.ts

```diff
@@ -0,0 +1,93 @@
+import { promises as fs } from 'fs';
+import { join } from 'path';
+import type { UserPreferences } from './preferences';
+
+const ENCODING_UTF8 = 'utf8';
```

Comment:
This can just be inlined, no?

Reply:
Yeah, it can be! Changing that to inline.